cli/cli.go

```diff
@@ -168,7 +169,11 @@ func Init() {
 			installCommand(args[0])
 		}
 	} else {
-		listCommand()
+		if fsutil.CheckPerms("FRS", VERSION_FILE) {
```

Comment:
I was sure there was a `list` command that was run by default 😕 The `listCommand` function name kinda suggests it.

Should I add the `--list, -l` switch that runs `listCommand`?